### PR TITLE
Legacy 301 fixes

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Legacy 301 fixes

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-service/2024-10-11-collection.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Automox C2C Source (Collection)
```
Comment (jpipkin1) [inline blog-service/2024-10-11-collection.md:2]: You need to remove this release note file dated 10-11. The release note was already published on 10-14: https://help.sumologic.com/release-notes-service/2024/10/14/collection/

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, I found one thing you need to fix.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/automox-source.md [hunk 2]
```diff
@@ -33,15 +33,15 @@ Automox is a cloud-based platform for automated IT operations, offering patching
 
 The Automox source requires you to provide the Bearer Token and Organization ID. Follow the below steps to generate the required values:
 
-- To generate the **Bearer Token**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-2-find-your-api-key-in-the-console). 
-- To generate the **Organization ID**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-3-create-your-first-api-call). 
+- To generate the **Bearer Token**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-2-find-your-api-key-in-the-console).
+- To generate the **Organization ID**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-3-create-your-first-api-call).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/automox-source.md:37]:
```suggestion
- To generate the **Bearer Token**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-2-find-your-api-key-in-the-console). 
- To generate the **Organization ID**, follow the instructions mentioned in the [Automox documentation](https://developer.automox.com/developer-portal/newbie-api-guide/#step-3-create-your-first-api-call). 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/automox-source.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ The Automox source requires you to provide the Bearer Token and Organization ID.
 When you create a Automox Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Create a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
 
 To configure a Automox Source:
-1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. 
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/automox-source.md:44]:
```suggestion
1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. 
```

File: blog-service/2024-10-11-collection.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2024-10-11-collection.md:2]: whoa, great catch! It must've gotten picked up in the merge with main

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce!

**Pull request merged**